### ISSUE TITLE
Brace delimited parameter in action

### ISSUE THREAD

**Issue by muzimuzhi** [blefloch/latex-unravel#60]
```tex
\documentclass{article}
\usepackage{unravel}

\def\mycmd#1#{}
\unravel{\mycmd arg{}}

\begin{document}
\end{document}
```

- Expected:
  ```
  [===== Step 1 =====] \mycmd = macro:#1#{->{}
  ```
- Actual:
  ```
  [===== Step 1 =====] \mycmd = macro:#1{->{
  ```

Note the trailing `#` in parameter text and ending `}` for replacement text are both missing.

```
||
|> \mycmd arg{}

[===== Step 1 =====] \mycmd = macro:#1{->{
||
|> {}

[===== Step 2 =====]
<| {
||
|> }

[===== Step 3 =====]
<| {}
||
|>

[===== End =====]
```

**Comment by blefloch** [comment]
Thanks for the many relevant issues you raised and sorry that they lingered for so long.

For this one, I don't see why you would expect `macro:#1#{->{}`.  In fact, I simply use the output of `\meaning\mycmd` here, which is `macro:#1{->{`, which seems fine: it means that `#1` is delimited by a left brace and the brace is left in the input stream.  So I'm tempted to close it as not an issue, do you agree?

On the other hand, these types of macros are problematic for another reason as they break an optimization, see #61, thanks for pointing me to this case.

**Comment by muzimuzhi** [comment]
> In fact, I simply use the output of `\meaning\mycmd` here, which is `macro:#1{->{`, which seems fine: it means that `#1` is delimited by a left brace and the brace is left in the input stream.

You're right. I totally forgot the form of output of `\meaning\mycmd` here.
